packages/gatsby-plugin-feed/src/gatsby-node.js

```diff
@@ -35,9 +40,14 @@ exports.onPreBootstrap = async function onPreBootstrap(
   delete pluginOptions.plugins
 
   try {
-    const normalized = await pluginOptionsSchema.validate(pluginOptions)
+    // TODO: remove this once pluginOptionsSchema is stable
+    const normalized = await pluginOptionsSchema({ Joi }).validate(
```

Comment:
I'm not sure to understand why we need to but this in preBootStrap before it's getting stable 🤔

Reply:
This was already there. Removing it would make the current experience worse than before!